packages/client/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Change this to match your project
+  "include": ["*.js"],
+  "compilerOptions": {
+    // Tells TypeScript to read JS files, as
+    // normally they are ignored as source files
+    "allowJs": true,
+    // Generate d.ts files
+    "declaration": true,
+    // This compiler run should
+    // only output d.ts files
+    "emitDeclarationOnly": true,
+    // Types should go into this directory.
+    // Removing this would place the .d.ts files
+    // next to the .js files
+    "outDir": "lib",
```

Comment:
We should make this `dist` whenever possible.

Reply:
will update it!